src/sardana/pool/poolcontrollers/DiscretePseudoMotorController.py

```diff
@@ -38,39 +38,56 @@
 
 CALIBRATION = 'Calibration'
 LABELS = 'Labels'
+MSG_API = 'Configuration attribute is in use. Labels and Calibration ' \
+          'attributes are deprecated since version Jul18'
 
 
 class DiscretePseudoMotorController(PseudoMotorController):
-    """A discrete pseudo motor controller which converts physical motor positions
-    to discrete values"""
+    """
+    A discrete pseudo motor controller which converts physical motor
+    positions to discrete values"""
 
     gender = "DiscretePseudoMotorController"
     model = "PseudoMotor"
     organization = "Sardana team"
     image = ""
 
-    pseudo_motor_roles = ("OutputMotor",)
```

Comment:
IMHO neither the previous names nor the new ones are self explanatory. And what about ContinuousMoveable and DiscreteMoveable?

Reply:
We have adopted the proposed names.